src/Compilers/CSharp/Portable/Compilation/MemberSemanticModel.cs

```diff
@@ -1968,15 +1978,13 @@ BoundNode bind(CSharpSyntaxNode root, out Binder binder)
             void rewriteAndCache()
             {
                 var diagnostics = DiagnosticBag.GetInstance();
-#if DEBUG
```

Comment:
I removed this if-directive because I didn't see a clear reason to *not* do this bailout in release mode. When this directive is left in, we get the following test failure:

```
Failed Microsoft.CodeAnalysis.CSharp.UnitTests.NullablePublicAPITests.NullableDisableSemanticModel_01(runNullableAnalysisAlways: True) [7 ms]
  Error Message:
   Assert.Equal() Failure
Expected: None
Actual:   Annotated
  Stack Trace:
     at Microsoft.CodeAnalysis.CSharp.UnitTests.NullablePublicAPITests.<NullableDisableSemanticModel_01>g__test|116_0(SemanticModelOptions options, NullableAnnotation expectedAnnotation, <>c__DisplayClass116_0&) in C:\Users\rigibson\src\roslyn\src\Compilers\CSharp\Test\Symbol\Symbols\Source\NullablePublicAPITests.cs:line 5200
   at Microsoft.CodeAnalysis.CSharp.UnitTests.NullablePublicAPITests.NullableDisableSemanticModel_01(Boolean runNullableAnalysisAlways) in C:\Users\rigibson\src\roslyn\src\Compilers\CSharp\Test\Symbol\Symbols\Source\NullablePublicAPITests.cs:line 5190
```

Basically, we want a semantic model which "disables nullable analysis" to return nullable-disabled results, even if feature flags indicate nullability analysis should be performed and thrown away.